src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RefreshOrchestrator.java

```diff
@@ -114,7 +111,7 @@ public void validate(Long jobId, UserRequest userRequest) {
     }
 
 
-    private ResponseEntity<Object> refreshJobByServiceName(Map<String, HttpStatus> responseCodeWithUserId,
+    protected ResponseEntity<Object> refreshJobByServiceName(Map<String, HttpStatus> responseCodeWithUserId,
```

Comment:
required to be package private rather than class level so I could add proper coverage.